docs/mui-vale/styles/MUI/MuiBrandName.yml

```diff
@@ -3,7 +3,7 @@
 # For example, Apple does this meticulously with their brand name: https://www.apple.com/macbook-air/.
 # Also read https://www.chrisdpeters.com/blog/using-non-breaking-spaces-to-help-with-branding/ for why.
 extends: substitution
-message: Use a non-breaking space for brand name ('%s' instead of '%s')
+message: Use a non-breaking space (option + space, instead of space) for brand name ('%s' instead of '%s')
```

Comment:
Do we have folks using Windows? Then that'll also requir windows specific shortcut.

Reply:
Nice catch, let me add it.